torch_xla/csrc/tensor.cpp

```diff
@@ -1008,20 +1008,19 @@ void XLATensor::elu_(XLATensor& input, at::Scalar alpha, at::Scalar scale,
   input.SetIrValue(ir::ops::Elu(input.GetIrValue(), alpha, scale, input_scale));
 }
 
-XLATensor XLATensor::conv2d(
-    const XLATensor& input, const XLATensor& weight, const XLATensor& bias,
-    tensorflow::gtl::ArraySlice<const xla::int64> stride,
-    tensorflow::gtl::ArraySlice<const xla::int64> padding) {
+XLATensor XLATensor::conv2d(const XLATensor& input, const XLATensor& weight,
+                            const XLATensor& bias,
+                            std::vector<xla::int64> stride,
+                            std::vector<xla::int64> padding) {
   ir::NodePtr ir_value =
       ir::MakeNode<ir::ops::Conv2d>(input.GetIrValue(), weight.GetIrValue(),
                                     bias.GetIrValue(), stride, padding);
   return input.CreateFrom(ir_value);
 }
 
-XLATensor XLATensor::conv2d(
-    const XLATensor& input, const XLATensor& weight,
-    tensorflow::gtl::ArraySlice<const xla::int64> stride,
-    tensorflow::gtl::ArraySlice<const xla::int64> padding) {
+XLATensor XLATensor::conv2d(const XLATensor& input, const XLATensor& weight,
+                            std::vector<xla::int64> stride,
+                            std::vector<xla::int64> padding) {
   ir::NodePtr ir_value = ir::MakeNode<ir::ops::Conv2d>(
       input.GetIrValue(), weight.GetIrValue(), stride, padding);
```

Comment:
If you do not move the values, this is kinda pointless :)

Reply:
Done.